cmd/dep/status_test.go

```diff
@@ -305,6 +305,7 @@ func TestCollectConstraints(t *testing.T) {
 	ver1, _ := gps.NewSemverConstraintIC("v1.0.0")
 	ver08, _ := gps.NewSemverConstraintIC("v0.8.0")
 	ver2, _ := gps.NewSemverConstraintIC("v2.0.0")
+	master := gps.NewBranch("master")
```

Comment:
This can be used inline. Other's are not inline because they do not return single value.